docs/consent/README.md

```diff
@@ -0,0 +1,38 @@
+<h1 align="center"><img src="https://make.hmn.md/altis/Altis-logo.svg" width="89" alt="Altis" /> Consent</h1>
+
+<p align="center">Enhanced user privacy and consent controls for <strong><a href="https://altis-dxp.com/">Altis</a></strong>.</p>
+
+<p align="center"><a href="https://packagist.org/packages/altis/consent"><img alt="Packagist Version" src="https://img.shields.io/packagist/v/altis/consent.svg"></a></p>
+
+## What is the Altis Consent module?
+
+Altis Consent module provides a website administrator(you) with a centralized and consistent way of obtaining data privacy consent from their website's users/visitors. Data privacy consent refers to how user data is tracked, collected, or stored on a website.
+
+### Web Cookies
+A website uses cookies to track user behaviour on the website. Web cookies can be of three types: session, persistent and third party. Depending on their function, these cookies can further be categorized into functional or operational cookies, marketing or personalized cookies, third party cookies and their sub-categories. While functional cookies are necessary for a smooth user experience, other cookies, such as marketing cookies that track user preferences, are not essential for a website's performance. General Data Protection Regulation (GDPR) governs user data privacy and protection and stipulates that businesses require appropriate consent from website visitors before they are served any such cookies.
```

Comment:
> Web cookies can be of three types: session, persistent and third party

Was this based on some other source?  1st party and 3rd party cookies aren't necessarily persistent, right? I think we might want to simplify this down a bit because 3rd party cookies could technically fall into almost any category if we define those as cookies from a different host / domain.

Maybe we just say there are 2 broad types, 1st party cookies set by the server or by code on the domain itself and 3rd party cookies set by scripts included from other domains or services such as Google Analytics. These can then be further categorized as functional, marketing etc... etc...